.travis.yml

```diff
@@ -139,6 +134,10 @@ before_install:
   - df -h
   - ulimit -a
 install:
+- export NINJA_URL="https://github.com/Kitware/ninja/releases/download/v1.7.2.gaad58.kitware.dyndep-1/"
+- export NINJA_TARBALL_NAME="ninja-1.7.2.gaad58.kitware.dyndep-1_x86_64-linux-gnu"
+- curl -L ${NINJA_URL}${NINJA_TARBALL_NAME}.tar.gz | tar xz
+- export PATH=$PWD/${NINJA_TARBALL_NAME}:$PATH
```

Comment:
looks like you can `conda install ninja -c conda-forge` for 1.7.2

Reply:
True, thanks for pointing that out. Unfortunately they ship the official v1.7.2, while only the Kitware-maintained fork supports Fortran.